mensor/measures/types.py

```diff
@@ -446,10 +448,7 @@ def transforms(self):
     @transforms.setter
     def transforms(self, transforms):
         # TODO: Check structure of transforms dict
-        if not transforms:
```

Comment:
nonsubstantive, but I do like oneliners

Reply:
Aye... I like this better too. I think I left it uncollapsed because I was expecting to do more logic than would be elegant in a one-liner. For now, this works well.